src/backend/distributed/utils/connection_cache.c

```diff
@@ -30,8 +30,8 @@
 #include "utils/palloc.h"
 
 
-/* state needed to prevent new connections during modifying transactions */
-bool IsModifyingTransaction = false;
+/* state needed to keep track of operations used during a transaction */
+XactModificationType XactModification = XACT_MODIFICATION_NONE;
```

Comment:
On 2016-08-30 15:00:16 -0700, Jason Petersen wrote:
> I was going to suggest `XactModificationType`, but that's the type name. Bah!

We really should stop using uppercase for global vars.